### ISSUE TITLE
Get CAPI queries working when on-foot

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#955]
This check in companion.py thinks the server is lagging:

```python
            elif ((auto_update and not data['commander'].get('docked'))
                  or (data['lastSystem']['name'] != monitor.system)
                  or ((data['commander']['docked']
                       and data['lastStarport']['name'] or None) != monitor.station)
                  or (data['ship']['id'] != monitor.state['ShipID'])
                  or (data['ship']['name'].lower() != monitor.state['ShipType'])):
                raise companion.ServerLagging()
```
There's suit/loadout stuff in the CAPI `/profile` data, so this does need to work.  Might need to implement a new 'on foot' monitor flag or somesuch.

**Comment by Athanasius** [comment]
Ah for one thing plugin journal_entry is being passed `station = None`, so I guess lastStarport isn't coming through ?

**Comment by Athanasius** [comment]
Oh and on-foot `Location` event has `"Docked":false` which won't help.

**Comment by Athanasius** [comment]
```
auto_update = False
data['commander'].get('docked') = False
data['lastSystem']['name'] = 'Adityan'
monitor.system = 'Adityan'
data['lastStarport']['name'] = 'Hopkins City'
data['ship']['id'] = 0
data['ship']['name'] = 'SideWinder'
```
So it's just the docked status blocking that I think.  Of course then we should probably be checking `data['suit']` if not docked and that we're otherwise on foot.  No doubt downstream code will need its assumptions checking too.

**Comment by Athanasius** [comment]
```
monitor.system = 'Adityan'
monitor.station = None
monitor.state['ShipID'] = None
monitor.state['ShipType'] = None
```

**Comment by Athanasius** [comment]
Fixed for 5.0.0-beta3.
